main.go

```diff
@@ -236,6 +237,12 @@ func main() {
 	}
 	testCtx, cancelFunc := context.WithTimeout(context.Background(), globalTimeout)
 
+	pauseBetweenMessages, err := time.ParseDuration(*argPauseBetweenMessages)
+	if err != nil {
+		fmt.Printf("Failed parse '--pause-between-messages': %q is not a valid duration string. See https://golang.org/pkg/time/#ParseDuration for valid duration strings\n", *argPauseBetweenMessages)
```

Comment:
To be consistent, could you please write the error to `os.Stderr` via `fmt.Fprintf`?

Reply:
Yes, sure can. I copied from the example above it, so I'll change that too.